### DISCUSSION TITLE
Assembla links not working

### DISCUSSION THREAD

**Discussion by martynakielbasa** [PlusToolkit/PlusLib#993] [General]
Hello, 

While trying to build fCal phantom 2.0, I noticed that all your assembla links containing the phantom model, screenshots, configuration files etc. are not working anymore. I get '404 page not found' error. 

I managed to find the library of all 3D models at another link, but going through your tutorial on building the phantom I feel like there is lots of useful data at the assembla links not available at the moment. 

Is there a chance you could help with this problem?

Kind regards
Martyna

**Comment by adamrankin** [comment]
Hi Martyna,

Could you send us the URL of the tutorial?

**Reply by martynakielbasa** [reply]
Of course. 

https://github.com/PlusToolkit/PlusDoc/raw/master/tutorials/PlusTutorialBuildingfCalPrintedPhantom.pptx

**Comment by adamrankin** [comment]
I have updated the documentation

**Reply by martynakielbasa** [reply]
Thank you!
